src/bin/cargo-flamegraph.rs

```diff
@@ -61,6 +61,10 @@ struct Opt {
     #[structopt(short = "f", long = "features")]
     features: Option<String>,
 
+    /// Open the output .svg file with default program
+    #[structopt(short = "O", long = "open")]
```

Comment:
Does `-O` have precedence in other tools? Cargo only allows `--open`, and I think we should do the same here unless there's a good reason to prefer `-O`.